### DISCUSSION TITLE
Profiles/Presets - I want a UI interface so i dont have to go into config

### DISCUSSION THREAD

**Discussion by BluthundAU** [cafali/SnapKey#104] [New Features & Ideas]
I want a UI interface so i don't have to go into config

May i ask why u haven't made that yet

**Comment by minteeaa** [comment]
i'd say the config is small and simple enough that a ui would be more overhead than is really necessary - but it isn't a bad idea, really. potentially could tinker with it later

**Reply by BluthundAU** [reply]
thx understanding my idea reason i want it the UI idea is it make it easier to change the keys use

**Reply by BluthundAU** [reply]
Thx 😊 🙏

**Comment by cafali** [comment]
I’m not the best at programming UI in apps, so I’ll pass on creating one… :p However, the implementation of the [1.2.9](https://github.com/cafali/SnapKey/releases/tag/v1.2.9) functionality offers almost the same benefits when it comes to user comfort and key rebinding.
I’ll mark this feature as implemented.

Enjoy!

**Reply by BluthundAU** [reply]
thx
